api/app/App.js

```diff
@@ -288,33 +288,79 @@ module.exports = function() {
             var exportType = req.body.exportType;
             var customHelper = require('../../helpers/custom.js');
             var accessList = customHelper.getAccessList(req)
-            if(!appKey){
+            if (!appKey) {
                 res.status(400).send("key is missing");
             }
-            if(!appId){
-                 res.status(400).send("appId is missing");
+            if (!appId) {
+                res.status(400).send("appId is missing");
             }
-            
-            if(!tableName){
-                 res.status(400).send("tableName is missing");
+
+            if (!tableName) {
+                res.status(400).send("tableName is missing");
             }
-            if(!exportType){
-                 res.status(400).send("exportType is missing");
+            if (!exportType) {
+                res.status(400).send("exportType is missing");
             }
-            global.appService.isMasterKey(appId,appKey).then(function(isMasterKey) {         
-                global.appService.exportTable(appId,tableName,exportType.toLowerCase(),isMasterKey,accessList).then(function(data) {
-                    if(exportType.toLowerCase()==='json'){
-                        res.status(200).json({data});
-                    }else{ res.status(200).send(data);}
-                }, function(err) {
-                        console.log("Error : Exporting Table.");
-                        console.log(err);
-                        res.status(500).send(err);
+            global.appService.isMasterKey(appId, appKey).then(function (isMasterKey) {
+                global.appService.exportTable(appId, tableName, exportType.toLowerCase(), isMasterKey, accessList).then(function (data) {
+                    if (exportType.toLowerCase() === 'json') {
+                        res.status(200).json({ data });
+                    } else { res.status(200).send(data); }
+                }, function (err) {
+                    console.log("Error : Exporting Table.");
+                    console.log(err);
+                    res.status(500).send(err);
                 });
-            }, function(error) {
+            }, function (error) {
+                return res.status(500).send('Cannot retrieve security keys.');
+            });
+
+        } catch (e) {
+            console.log(e);
+        }
+    });
+    // Import Tables
+    global.app.post('/import/:appId/:tableName', function (req, res) {
+        console.log("++++ Import Database ++++++");
+        try {
+            var appKey = req.body.key;
+            var appId = req.params.appId;
+            var tableName = req.params.tableName;
+            var importType = req.body.importType;
+            var accessList = customHelper.getAccessList(req)
+            if (!appKey) {
+                res.status(400).send("key is missing");
```

Comment:
Say, App Key is missing. Please have either Client Key or the Master Key in request body.